examples/e2e/cluster_group/README.md

```diff
@@ -0,0 +1,38 @@
+# Virtual Cluster demo
+
+End-to-end example of provisioning a Palette Virtual Cluster with all of its dependencies. This terraform configuration will provision the following resources on Spectro Cloud:
+- K8s host cluster and datasource.
+- Addon Cluster Profile
+
+## Instructions:
+
+Clone this repository to a local directory, and then change directory to `examples/e2e/virtual`. Proceed with the following:
+1. Provision host cluster as a prerequisite to use virtual cluster. 
+To achieve it add host configuration block to existing cluster or provision it from scratch using one of existing examples:
+<pre>
+  host_config {
+    host_endpoint_type = "LoadBalancer" 
+    ingress_host       = "*.dev.spectrocloud.com"
+  }
+</pre>
+2. From the current directory, copy the template variable file `terraform.template.tfvars` to a new file `terraform.tfvars`.
+3. Specify and update all the placeholder values in the `terraform.tfvars` file.
+4. Initialize and run terraform: `terraform init && terraform apply`.
+5. Wait for the cluster creation to finish.
+
+Once the cluster is provisioned, the cluster _kubeconfig_ file is exported in the current working directly.
+
+Export the kubeconfig and check cluster pods:
+
+```shell
+export KUBECONFIG=kubeconfig_eks
+kubectl get pod -A
+```
+
+## Cleanup:
```

Comment:
```suggestion
## Clean Up
```
Clean up is two words when it's an action (verb), but if you are talking about the act of cleaning such as "the cleanup yesterday took forever" then it's one word 😄

Reply:
@nikchern this one got missed.